rust/src/lib.rs

```diff
@@ -126,6 +126,9 @@ pub use protocol::checkpoints;
 #[cfg(feature = "integration_test")]
 pub mod test_utils;
 
+#[cfg(feature = "parquet")]
+const NULL_PARTITION_VALUE_DATA_PATH: &str = "__HIVE_DEFAULT_PARTITION__";
```

Comment:
why does this need to be defined in the top level?

Reply:
This const used to be defined in `src/writer/utils.rs`. I moved it to `src/lib.rs` because it is also used in `operations/convert_to_delta`. IMO it doesn't make a lot of sense to make `convert_to_delta` depend on `writer/utils.rs`, thus I defined this const in the top level. Do you have any suggestions on where to place this definition?

Reply:
Moved `NULL_PARTITION_VALUE_DATA_PATH` to `crate::schema::partitions`